conda/plugins/subcommands/doctor/__init__.py

```diff
@@ -10,7 +10,7 @@
 import argparse
 
 from ....base.context import context
-from ....cli.conda_argparse import (
+from ....cli.helpers import (
```

Comment:
Only the `add_*` functions are in `conda.cli.helpers`, `ArgumetnParser` is still defined in `conda.cli.conda_argparser`

Reply:
The tests pass because `conda.cli.helpers` is importing `argparse.ArgumentParser` for type annotation purposes